hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/filedrops/RequestFileDropGenerater.java

```diff
@@ -22,16 +23,24 @@
 
 	@Handler
 	public void createFileDrops(Exchange exchange) {		
+		String methodName = LoggingUtil.getMethodName();
+		logger.debug("Beging {}", methodName);
```

Comment:
Typo here. Also I think most of the log statements start with "{} -" so that the methodName is the first part of the statement.